UnityProject/Assets/Scripts/Mobs/Equipment/Equipment.cs

```diff
@@ -186,90 +186,118 @@ public string GetPlayerNameByEquipment()
 
 	public string Examine()
 	{
-		string theyPronoun = TheyPronoun;
-		theyPronoun = theyPronoun[0].ToString().ToUpper() + theyPronoun.Substring(1);
-		string theirPronoun = TheirPronoun;
+
 		string equipment = "";
+		string pronounIs = "";
+		string pronounHas = "";
+		string theyPronoun = "";
+		string theirPronoun = "";
+
+		if (GetPlayerNameByEquipment() == "Unknown" && IsIdentityObscured())
```

Comment:
this should probably be moved to a Utils class where you can get all this stuff and use from anywhere.